docs/development/local-development.md

```diff
@@ -81,6 +81,15 @@ Then you can run `pnpm` directly from Docker, for instance:
 docker run -it --rm -v "$PWD":/usr/src/app -w /usr/src/app renovatebot_local pnpm install
 ```
 
+## Run GitHub Action build job locally with act
+
+1. Install act - [installation instructions](https://github.com/nektos/act#installation)
+1. Run (Pick the Large image)
```

Comment:
why do we need the large image?

Reply:
I don't remember now why the large image is required. I do remember that the smaller image doesn't work.